fixtures/large/rspec_mocks_proxy_expected.rb

```diff
@@ -201,7 +201,9 @@ def message_received(message, *args, &block)
           expectation.unadvise(messages_arg_list)
           expectation.invoke(stub, *args, &block)
         elsif (expectation = find_almost_matching_expectation(message, *args))
-          expectation.advise(*args) if null_object? unless expectation.expected_messages_received?
```

Comment:
it is correct for standard to make this change, it's not a change rubyfmt can make